llvm/test/DebugInfo/NVPTX/debug-bool-var.ll

```diff
@@ -0,0 +1,84 @@
+; RUN: llc < %s -mtriple=nvptx64-nvidia-cuda | FileCheck %s
```

Comment:
Could we use -print-after=finalize-isel to reduce the surface area being tested (and it'd make the output slightly easier to read IMO)?

Reply:
Done. Test is reworded with "finalize-isel".